nx-libs.spec

```diff
@@ -804,23 +771,6 @@ ln -s -f ../../../../%{_lib}/libXext.so.6 %{buildroot}%{_libdir}/nx/X11/Xinerama
 %defattr(-,root,root)
 %{_libdir}/libNX_Xau.so.6*
 
-%files -n libNX_Xcomposite1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xcomposite.so.1*
-
-%files -n libNX_Xdamage1
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdamage.so.1*
-
-%files -n libNX_Xdmcp-devel
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so
-%{_includedir}/nx/X11/Xdmcp.h
-
-%files -n libNX_Xdmcp6
-%defattr(-,root,root)
-%{_libdir}/libNX_Xdmcp.so.6*
-
```

Comment:
The preferred way for SUSE seems to be leaving the old package around. It will be automatically removed on dist-upgrades if the new dist version doesn't provide a package anymore.

So, I propose the following: completely remove the `libnx-x{damage,composite,dmcp,tst}*` packages from the `nx-libs.spec` file and for Debian keep the dummy package change you have proposed, BUT also add `libxcomposite1` etc. dependencies to `Depends:` as mentioned on https://wiki.debian.org/Renaming_a_Package#Method_2